autogen/agentchat/contrib/function_store/function_store_utils.py

```diff
@@ -0,0 +1,19 @@
+import subprocess
+import sys
+import functools
+
+
+def requires(*packages):
+    def decorator(func):
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+            for package in packages:
+                try:
+                    __import__(package)
```

Comment:
This only works if the python package name is the same as the pip package name right? This can be broken by, for example "autogen" (install vis pyautogen). Or "PIL" (installed by pillow). Etc.

Reply:
Good catch! I'll improve the decorator. I don't want to create requirement files for possible 1000 functions. But I can add version and pip package name to the function!

Reply:
fixed.